### PR TITLE
Add readme to dashboard and database : Alex

### LINKED ISSUE #42: Create README.md for dashboard folder
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

Should contain:
- [ ] - What the dashboard shows
- [ ] - (Wireframe diagram)
- [ ] - How to access the dashboard
- [ ] - Other details as required

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #42  and #82 

I have done the following:
- Created README for database folder
- I have included standardised outline with brief synopsis
- Created README for dashboard folder
- I have included standardised outline with brief synopsis

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# StarWatch Dashboard
```
Comment (CalebMathers) [inline dashboard/README.md:1]: Lovely use of markdown headings

File: dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+This folder contains the scripts to run and dockerise the dashboard for the StarWatch pipeline project. The pipeline was created using `streamlit` and queries data from a postgres database using `psycopg2`.
```
Comment (CalebMathers) [inline dashboard/README.md:2]: Having libraries in `these` are good visually

File: dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Design
+
+
+__Figure 1__ - _Starwatch Dashboard Wireframe_: Shows the design for the StarWatch dashboard.
```
Comment (CalebMathers) [inline dashboard/README.md:7]: Image in the readme, informative and eye catching

File: database/README.md
Comment (CalebMathers) [inline database/README.md:1]: Same as comments for the other file, good start to the README

File: dashboard/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## Setup
+
+## Usage
```
Comment (CalebMathers) [inline dashboard/README.md:11]: Good headings

**Review by CalebMathers — APPROVED** [review]

File: dashboard/README.md
Comment (olliejth) [inline dashboard/README.md:1]: We used emojis on the title last week and it looked quite good?
Written in format => :emoji_name:

Comment (AlexOBarnes) [reply dashboard/README.md:1]: This is a good idea, I will open an issue for this

**Review by olliejth — APPROVED** [review]
Some minor points for formatting but otherwise good

File: dashboard/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# StarWatch Dashboard
```
Comment (MurasatoNaoya) [inline dashboard/README.md:1]: Good use of markdown headings to have clear delineations in the dashboard README.

File: dashboard/README.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+This folder contains the scripts to run and dockerise the dashboard for the StarWatch pipeline project. The pipeline was created using `streamlit` and queries data from a postgres database using `psycopg2`.
+
+## Design
+![ERD](../assets/starwatch_architecture_diagram.png)
```
Comment (MurasatoNaoya) [inline dashboard/README.md:5]: Good link to the ERD diagram, to showing what data the dashboard is pulling from.

**Review by MurasatoNaoya — APPROVED** [review]

File: dashboard/README.md (hunk 6 above)
Comment (Ruyzambrano) [reply dashboard/README.md:5]: You could also link in the streamlit and psycopg2 libraries here

**Pull request merged**